warehouse/data-dictionary-core/pom.xml

```diff
@@ -18,6 +22,11 @@
             <groupId>gov.nsa.datawave.microservice</groupId>
             <artifactId>dictionary-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax.enterprise</groupId>
+            <artifactId>cdi-api</artifactId>
+            <version>2.0.SP1</version>
```

Comment:
This version may not be strictly necessary

Reply:
it isn't (i think the ide was being helpful)